exercises/book-store/src/lib.rs

```diff
@@ -1,3 +1,3 @@
-pub fn lowest_price(_: &[usize]) -> usize {
-    unimplemented!()
+pub fn lowest_price(input: &[usize]) -> usize {
```

Comment:
`input` doesn't tell me what the meaning of the input is. I'd like to see `books`.